### DISCUSSION TITLE
Stellarium doesn't start/load

### DISCUSSION THREAD

**Discussion by BlackApron** [Stellarium/stellarium#2445] [Q&A]
Hi... I had to reinstall my computer. It is the same computer that previously ran Stellarium without issues. But after reinstalling win10 and installing Stellarium 0.22.1, it just doesn't start. ANGLE, MESA, all versions show the same messages in the log:

QOpenGLFramebufferObject: An undefined error has occurred:  1285
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete, missing attachment.

I don't know what else to try.  Stellarium shows the splash screen but then it just shows a black window that closes a few seconds after.

Any suggestions?

Thanks!

**Comment by gzotti** [comment]
Have you installed all relevant drivers for this computer? Esp. the Graphics card drivers? Do other GPU-intensive programs work?

**Reply by BlackApron** [reply]
Hi... yes... all drivers are up to date... and other programs such as TheSkyX or StarryNight 8 work perfectly fine.
![Display Driver](https://user-images.githubusercontent.com/22322561/168672949-06b9e42f-03db-490c-b75c-3406d32a94bd.jpg)
